x-pack/test_serverless/functional/test_suites/observability/cypress/cypress.config.ts

```diff
@@ -6,6 +6,15 @@
  */
 
 import { defineCypressConfig } from '@kbn/cypress-config';
+import { kbnTestConfig } from '@kbn/test';
+
+import Url from 'url';
+
+const kibanaUrl = Url.format({
```

Comment:
It could be just me but I prefer the specific name `kibanaUrlWithoutAuth` which we use in the stateful version. 
Apart from being consistent, it is self-explanatory and by reading the code you know what to expect. WDYT?